packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -140,7 +140,8 @@ class CodePushClient {
     final response = await _httpClient.send(request);
     final body = await response.stream.bytesToString();
 
-    if (response.statusCode != HttpStatus.ok) {
+    if (response.statusCode != HttpStatus.ok &&
```

Comment:
Might also be nice to have an extension on HttpStatus like `isCreatedOrOk` to eliminate some of the duplication and then we can leave a todo on the extension method to remove it once we've rolled out the backend changes and done a force upgrade in the future.

Reply:
I'd be curious to hear your thoughts re: having an even more general `isSuccess` extension, something like:

```dart
bool get isSuccess => statusCode >= 200 && statusCode < 300;
```

Reply:
Or maybe `< 400`, depending on our feelings about redirects.

Reply:
Went ahead and did this. It's a quite a bit less rigid than the previous implementation but I don't think we're losing anything in terms of correctness.